### PR TITLE
changed postgres port for local development

### PR DESCRIPTION
**Pull request by slustig210**
@kritisingh0114 this should fix the problem

defaults postgres port to 5433, but u can set it using the UREFER_DB_PORT

### CODE REVIEW

**Review by swjeong0825 — CHANGES_REQUESTED** [review]
I got this error when I ran this branch: 
image

I ran the docker successfully with the main branch, so I think my docker is supposed to work fine with this change.

File: docker-compose.yml [hunk 1]
```diff
@@ -41,11 +41,12 @@ services:
     container_name: urefer-db
     restart: always
     ports:
-      - 5432:5432
+      - ${UREFER_DB_PORT:-5433}:5432
```
[the remark this answers is not in the document]
Comment (swjeong0825) [reply docker-compose.yml:44]: @slustig210 Do you have an idea about this? it seems like the file is failing from reading out the DATABASE_URL env field.

**Comment by rdonati** [comment]
@swjeong0825 I got the same error. Turned out it was a single vs double quote thing in `run.sh`. Should be fixed now

**Comment by slustig210** [comment]
> Turned out it was a single vs double quote thing in `run.sh`.

Oops lol, gotta switch it in migrate reset then as well

**Review by Danny123234 — APPROVED** [review]

**Review by swjeong0825 — APPROVED** [review]
Everything looks good to me!

**Pull request merged**